js/src/rive_advanced.mjs.d.ts

```diff
@@ -20,7 +20,7 @@ interface RiveOptions {
     StrokeCap: typeof StrokeCap;
     StrokeJoin: typeof StrokeJoin;
   
-    load(buffer: Uint8Array): File;
+    load(buffer: Uint8Array): Promise<File>;
```

Comment:
Looks like the TS code was already "await"ing load, I just changed it to be a Promise.